http-custom-mechanism/custom-module/pom.xml

```diff
@@ -20,16 +20,12 @@
     <modelVersion>4.0.0</modelVersion>
     <parent>
         <groupId>org.wildfly.quickstarts</groupId>
-        <artifactId>wildfly-quickstart-parent</artifactId>
-        <!--
-        Maintain separation between the artifact id and the version to help prevent
-        merge conflicts between commits changing the GA and those changing the V.
-        -->
-        <version>2</version>
-        <relativePath/>
+        <artifactId>http-custom-mechanism</artifactId>
+        <version>27.0.0.Beta1-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
     </parent>
 
-    <artifactId>http-custom-mechanism</artifactId>
+    <artifactId>custom-module</artifactId>
```

Comment:
Why is this being renamed?

Reply:
Actually, just noticed that you already mentioned this in the description so you can ignore this comment :)